TAF/config/core-data/configuration.py

```diff
@@ -1,19 +0,0 @@
-LOG_LEVEL = "INFO"
```

Comment:
Since there's no file in core-data directory, is the core-data folder still needed?